userprofile/tests/test_views.py

```diff
@@ -190,7 +190,7 @@ def test_get_list(self):
             f"{urlencode({'timestamp_gt': triage_old.timestamp.isoformat()})}"
         )
 
-        r = dict(response.data[0])
+        r = dict(response.data["results"][0])
```

Comment:
Is this change because of the DEFAULT_PAGINATION_CLASS change in settings? I'm worried that it might break things down the line

Reply:
This is due to `PAGE_SIZE` setting. This view sets the pagination class, but because there was no page size set, it did nothing and there was no pagination on any of the views that requested it. I don't think anything is using this (we were supposed to move things off of ndoh hub here).

Reply:
I recon nothing is using the list views then, so don't think anything will break.